packages/babel-parser/src/plugins/flow.js

```diff
@@ -2135,6 +2135,9 @@ export default (superClass: Class<Parser>): Class<Parser> =>
         (code === charCodes.greaterThan || code === charCodes.lessThan)
       ) {
         return this.finishOp(tt.relational, 1);
+      } else if (this.state.inType && code === charCodes.questionMark) {
```

Comment:
Do we have tests for `??string`?

Reply:
We do, it was previously fixed in #8115. I forgot to mention that.

Reply:
Cool, just wanted to double check :)